### PR TITLE
Button and logo added to social media profile links.

### PR DESCRIPTION
**Pull request by adnanraghib1308**
To make the links more attractive added button and logo of social media in the social media links section.
See the screen shot

Screenshot 2020-12-26 at 9 25 44 AM

### CODE REVIEW

**Comment by adnanraghib1308** [comment]
@ankushdharkar please review this

File: index.html [hunk 1]
```diff
@@ -58,13 +58,36 @@ <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
...
+        <a class="linkedinlink" 
+        href="https://www.linkedin.com/company/real-dev-squad/" target="_blank">
+          <button class="linkedin-btn" >
+            LinkedIn
+            <img class="logo-linkedin" src = "img/linkedin512.png" alt="linkedinlogo">
+          </button></a>
         <span>|</span>
-        <a href="https://twitter.com/realdevsquad" target="_blank">Twitter</a>
+        <a class="twitterlink" 
+        href="https://twitter.com/realdevsquad" target="_blank">
```
Comment (sumitd94) [inline index.html:69]: Please add `rel="noreferrer"`

**Review by sumitd94 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (adnanraghib1308) [reply index.html]: @sumitd94 done

File: css/main.css [hunk 2]
```diff
@@ -131,11 +132,87 @@ nav .user-greet .user-greet-msg {
...
+.logo-instagram{
+  vertical-align: middle;
+  width: 25px;
+  height: 25px;
+}
+
+.instagramlink{
+  text-decoration: none;
+}
+
```
Comment (ankushdharkar) [inline css/main.css:208]: @sumitd94 How does this CSS look? Any issues?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
@sumitd94 Please check the tag and recommend next steps to @adnan1308

**Comment by ankushdharkar** [comment]
@adnanraghib1308 Facebook logo looks squished to me. Please fix its ratio.

Also, let's remove the border around the buttons please.

**Comment by adnanraghib1308** [comment]
@ankushdharkar done. Please see the screenshot for required changes.

File: css/main.css (hunk 2 above)
Comment (sumitd94) [reply css/main.css]: Mixture of `rem` and `px` ?

Comment (ankushdharkar) [reply css/main.css]: @sumitd94 No. Look closely again.

**Comment by ankushdharkar** [comment]
Thank you @adnanraghib1308. Why is the LinkedIn logo not the right color/fill, compared to the style of the other icons?

**Comment by Harshi7016** [comment]
I was just going through the css and observed text-decoration:none repeated 4 times with different social media links can't it be generalized ?
Just a thought and happy to be corrected

**Comment by Harshi7016** [comment]
And same for logo too

File: css/main.css (hunk 2 above)
Comment (swarajpure) [reply css/main.css]: @ankushdharkar same and repeated css for Facebook btn, Twitter btn, LinkedIn btn etc.

**Comment by adnanraghib1308** [comment]
@ankushdharkar @swarajpure @sumitd94 @Harshi7016 Made css more generalized, removed the repeated css for different social media buttons and links. Fixed the linkedin logo (see screenshot). Please let me know for any other improvement or changes. Thanks.

**Review by ankushdharkar — APPROVED** [review]

**Pull request merged**